nilearn/plotting/surf_plotting.py

```diff
@@ -343,7 +344,7 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     view: {'lateral', 'medial', 'dorsal', 'ventral', 'anterior', 'posterior'}, default is 'lateral'
         View of the surface that is rendered.
 
-    threshold : a number, None, or 'auto', default is None
+    threshold : a number or None, default is None
```

Comment:
Is "auto" now an invalid value? In which case, it should be mentioned in the whats_new.

Reply:
As far as I can see it was never a valid input, just a fluke in the docstring